pipelines/concourse.yml

```diff
@@ -482,10 +482,13 @@ jobs:
       trigger: true
     - get: unit-image
       tags: [k8s-topgun]
-    - get: charts
+    - get: concourse-chart
       passed: [k8s-smoke]
       tags: [k8s-topgun]
       trigger: true
+    - get: helm-charts
+      passed: [k8s-smoke]
```

Comment:
same as for `branch.yml` - this addition here will make setting the pipeline fail